network/netplay/netplay.c

```diff
@@ -965,9 +960,25 @@ static bool netplay_init_buffers(netplay_t *netplay, unsigned frames)
    if (!netplay->buffer)
       return false;
 
-   /* WORKAROUND: The code to initialize state buffers really should be here.
-    * It's been moved to work around cores that can't core_serialize_size
-    * early. */
+   {
+      unsigned i;
+      retro_ctx_size_info_t info;
+
```

Comment:
I left this here because the whole section is destined very soon to get lifted into a separate function. I can fix it if you'd like, but it certainly won't look like that in the quirks PR.